lib/helpbase.gi

```diff
@@ -211,7 +211,7 @@ if Length(positions) > 0 then # matches found
   for i in [1..Length(positions)] do
     Add( chop, [ topic{[begin..patterns[i].start-1 ]} ] );
     Add( chop, patterns[i].pattern );
-    begin := Minimum( patterns[i].finish+1, Length(topic) );
+    begin := Minimum( patterns[i].finish, Length(topic) )+1;
```

Comment:
Oops, as with #4213, the same change should be made to `hpcgap/lib/helpbase.gi`, please.